### DISCUSSION TITLE
Just two questions...

### DISCUSSION THREAD

**Discussion by webbasan** [dirkwhoffmann/vAmiga#795] [Q&A] [answered]
Hi,

since I couldn't find an email address to get in contact, I'll try it this way... :-)

I try to recreate my own Amiga history in virtual environments -- is there any chance that this might evolve one day to emulate an Amiga 3000? 
As far as I can see there is already some support for ECS and at least for the 68020, but the 3000 would require full support for the 68030/68882 (with embedded MMU)... But it would be nice to get AMIX up again... ;-)

And, since I'm also living in Karlsruhe: Are you interested in real hardware? I'm in search for a new home of my Amiga 1000 and stuff... :-)

**Comment by dirkwhoffmann** [comment, marked as answer]
> As far as I can see there is already some support for ECS and at least for the 68020, but the 3000 would require full support for the 68030/68882 (with embedded MMU)

I don't plan on AGA support (which isn't required for the A3000), but I haven't quite given up on plans to support an MMU as well as an FPU. Therefore, A3000 support may be coming at some point. Actually, I was very close to finishing the MMU. I got to the point where I needed to compare my implementation against real hardware. To do so, I bought two 68030 accelerator cards for my A500s to run some software, but both failed to run a simple test program which was originally written by David Hayne (it was the simplest test program I found, so it was the perfect test case to start with). Expecting a hardware issue, I contacted one of the board manufacturers, but they were not interested in even running the test program on their board. As a result, I lost confidence in modern accelerator cards and put the project on hold. 68882 support, on the other hand, is completely missing at the moment. I haven't even started working on it.

> And, since I'm also living in Karlsruhe: Are you interested in real hardware?

Supporting me with a real A1000 would be awesome. All my current test machines are A500’s (in different configurations), but I don’t have any machine with the old Agnus chip installed. You can contact me at dirk.hoffmann@h-ka.de.
